addons/udes_stock/models/stock_picking_type.py

```diff
@@ -435,6 +435,13 @@ class StockPickingType(models.Model):
         help="Enable/Disable Transfer count button on kanban view",
     )
 
+    # Enable printing of delivery notes
+    u_enable_delivery_note_printing = fields.Boolean(
+        string="Enable printing of delivery notes from the front end",
```

Comment:
Enable front end DN printing? to shorten it a bit. We have the help for longer description

Reply:
Text changed